src/main/ml-modules/root/com.marklogic.smart-mastering/constants.xqy

```diff
@@ -52,3 +52,7 @@ declare variable $TRACE-MATCH-RESULTS := "SM-MATCH";
 
 (: ERRORS :)
 declare variable $NO-MERGE-OPTIONS-ERROR := xs:QName("SM-NO-MERGING-OPTIONS");
+
+(: Scope for instance bodies :)
+declare variable $JSON-INSTANCE as xs:string? := "instance";
```

Comment:
Why the "?" on the types for this line and the next? `"instance"` is never going to fail to be a string